worker/buildbot_worker/commands/fs.py

```diff
@@ -235,7 +235,11 @@ def start(self):
         pathname = os.path.join(self.builder.basedir, self.args['path'])
 
         try:
-            files = glob.glob(pathname)
+            # recursive matching is only support in python3.5+
+            if sys.version_info[:2] >= (3, 5):
+                files = glob.glob(pathname, recursive=True)
```

Comment:
just saying (i'm not sure if it's easier to read): `files = glob.glob(pathname, recursive=True if sys.version_info[:2] >= (3, 5) else False)`

Reply:
The problem with your code is that python2 does not support `recursive` at all, you can't just pass `false` to it.

Reply:
right, this is the build worker code, sorry